fhttp/http_client.go

```diff
@@ -388,9 +391,20 @@ func NewStdClient(o *HTTPOptions) *Client {
 		},
 		TLSHandshakeTimeout: o.HTTPReqTimeOut,
 	}
-	if o.Insecure && o.https {
-		log.LogVf("using insecure https")
-		tr.TLSClientConfig = &tls.Config{InsecureSkipVerify: true} // nolint: gas
+	if o.https {
+		tr.TLSClientConfig = &tls.Config{}
+		if o.Insecure {
+			log.LogVf("using insecure https")
+			tr.TLSClientConfig.InsecureSkipVerify = true
+		}
+		if len(o.Cert) > 0 && len(o.Key) > 0 {
+			cert, err := tls.LoadX509KeyPair(o.Cert, o.Key)
+			if err != nil {
+				log.LogVf(fmt.Sprintf("%v", err))
```

Comment:
this probably should be a log.Err (maybe even fatal)

Reply:
got you, changed to log.Err.